pages/blog/hacktoberfest-2022.md

```diff
@@ -0,0 +1,207 @@
+
+## Why Hacktoberfest
+
+Hacktoberfest is an annual event that encourages people to contribute to open source throughout October. AsyncAPI is an open-source project and, like many other open-source projects, built and maintained by passionate people who often don't have the staff or budgets to do much more than keep the project alive. Hacktoberfest is all about giving back to those projects, sharpening skills, and celebrating all things open source, especially those who make open source so special.
+
+## Planting the seed
+
+For us, contributions come in all shapes and sizes. From typo fixes, bug reporting to more significant improvements. But that's not all! AsyncAPI also creates time to help onboard contributors that are finding it difficult to start contributing to the project or provide help to contributors that get stuck while contributing. 
+
+## How we prepared
+
+Not gonna lie, this year's Hacktoberfest snuck up on us like Batman in Gotham City, and as soon as we realized this, we immediately started planning. We kicked off by defining our goals for Hacktoberfest as we've always done. 
+
+- Promote [AsyncAPI Initiative](https://www.asyncapi.com/) as a place where we work not only on the AsyncAPI specification but also lots of tools
+- Help members of the broader open-source community make their first contributions in a friendly environment
+
+We spent time writing a how-to-contribute guide for folks participating for the first time and also wrote a maintainers guide for potential maintainers.
+
+AsyncAPI is a great project to contribute to because you can learn about AsyncAPI while fixing an issue. That's why we carefully curated issues and prepared especially for Hacktoberfest. And because most of these issues are existing issues, we made sure we reached out to project maintainers to identify which of them is a good one for Hacktoberfest contributors to look into
+
+## Was it worth it?
+
+Absolutely! Even despite the fact that we kicked off late, we still received an amazing contribution. AsyncAPI merged 18 PRs from 12 contributors as part of this year's Hacktoberfest! I know it's less than the previous years but let's look on the bright side because some of these contributions were fun to make.
+
+## Meet the heroes
+
+This is a huge amount of work, and we are really proud that so many wonderful people have come to help AsyncAPI to produce better software.
+
+We want to thank everyone personally who submitted improvements to our repositories. Here are the names of our heroes:
+
+<table>
+  <tr>
+    <td align="center">
+      <a href="https://github.com/Amishakumari544">
+        <img src="https://avatars.githubusercontent.com/u/49230384?v=4" width="100px;" alt=""/>
+        <br />
+        <sub>
+          <b>
+            Amishakumari544 -  1 PR
+          </b>
+        </sub>
+      </a>
+    </td>
```

Comment:
I don't wanna make review for md content (please wait for @alequetzalli), but maybe will be better to create React component for that, hmm, wdyt? 🤔

Reply:
separate PR is the biggest lie of a person on open source 😄 `I'll fix it right away but in separate PR` and PR will never be created XD That's only mu suggestion, you can create or not if you don't want. I don't have a problem with it.

Reply:
I suggested this because I also have a post on the conference where I'll be giving a special shoutout to the volunteers and I'd eventually be listing them like the one above... So the PR will eventually be created 😄

Reply:
OK, so PR will be created, nice 😄